art/step/steps.py

```diff
@@ -78,7 +81,7 @@ class OverfitOneBatch(ModelStep):
     def __init__(
         self,
         model: ArtModule,
-        number_of_steps: int = 100,
+        number_of_steps: int = 50,
```

Comment:
Okay, but does it matter?

Reply:
Oh I changed it for the purpose of new tutorial. Actually Idk what default value is the best